framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2430,19 +2426,6 @@ private void logBugInCF(BugInCF ce) {
         printMessage(msg.toString());
     }
 
-    /**
-     * Side-effects the array to make each string lowercase, then returns the array.
-     *
-     * @param a an array of strings
-     * @return {@code a}, but each element has been lowercased
-     */
-    private static String[] arrayToLowerCase(String[] a) {
```

Comment:
The use of `Arrays.setAll` and a lambda is shorter than this implementation and could be used here.
My preference would be to retain the `arrayToLowerCase` abstraction, though, because its purpose is obvious from its name, whereas I had to read the `Arrays.setAll` line each time I saw it to understand what it does.  Using the method is also shorter in the client.  Defining the method takes more characters than are saved at the client, but it's more important that clients are short.

Alternately, we could remove the undocumented behavior of lowercasing user-written `@SuppressWarnings` for the user.